x-pack/plugins/triggers_actions_ui/public/application/sections/alerts_table/alerts_flyout/alerts_flyout.test.tsx

```diff
@@ -6,10 +6,9 @@
  */
 import React from 'react';
 import { mountWithIntl, nextTick } from '@kbn/test-jest-helpers';
-import { EcsFieldsResponse } from '@kbn/rule-registry-plugin/common/search_strategy';
 import { act } from 'react-dom/test-utils';
 import { AlertsFlyout } from './alerts_flyout';
-import { AlertsField } from '../../../../types';
+import { Alert, AlertsField } from '../../../../types';
```

Comment:
I created the following types

```
export type Alert = EcsFieldsResponse;
export type Alerts = Alert[];
```

and use them everywhere. I think is clearer.